tests/commands/test_unset.py

```diff
@@ -8,32 +8,31 @@
          'lap top_ap ple_mac book pro']
 
 directories = ['.',
-               'simple',
                's p a c e s',
-               's p a/c e s',
                'r/e/c/u/r/s/i/v/e',
                'overlap/one',
                'overlap/two',
-               'very/very/very/deep']
+               'very/very/very/deep',
+               ]
 
 assets = [f"{d}/{f}.{i}" for f in files for i, d in enumerate(directories)]
 
+content_dict = {"one": "value",
+                "two": "input",
+                "three": "field"}
 
-@pytest.mark.repo_files(*assets)
+content_str = "\n".join([f"{elem}: {content_dict.get(elem)}"
+                         for elem in content_dict]) + "\n"
+
+contents = [[x, content_str] for x in assets]
+
+@pytest.mark.repo_contents(*contents)
 @pytest.mark.parametrize('asset', assets)
 def test_unset(repo: Repo, asset: str) -> None:
     """
     Test that `onyo unset KEY <asset>` removes keys from of assets.
     """
-    set_values = "key=value"
-    key = "key"
-
-    # TODO: find out if there is a faster way than `onyo set` for writing
-    # without leaving an unclean git tree
-    ret = subprocess.run(['onyo', 'set', '--yes', '--keys', set_values, '--path', asset], capture_output=True, text=True)
-    assert ret.returncode == 0
-    assert "key: value" in Path.read_text(Path(asset))
-
+    key = list(content_dict.keys())[0]
```

Comment:
As the key is still hard-coded later, I recommend the following as more readable and explicit:
```
key = content_dict.get('one')
```

Reply:
This would get the value.